src/napari_micromanager/_mda_handler.py

```diff
@@ -107,15 +111,20 @@ def _on_mda_started(self, sequence: MDASequence) -> None:
         # determine the new layers that need to be created for this experiment
         # (based on the sequence mode, and whether we're splitting C/P, etc.)
         axis_labels, layers_to_create = _determine_sequence_layers(sequence)
+
         yx_shape = [self._mmc.getImageHeight(), self._mmc.getImageWidth()]
 
         # now create a zarr array in a temporary directory for each layer
         for id_, shape, kwargs in layers_to_create:
             tmp = tempfile.TemporaryDirectory()
             dtype = f"uint{self._mmc.getImageBitDepth()}"
-
             # create the zarr array and add it to the viewer
-            z = zarr.open(str(tmp.name), shape=shape + yx_shape, dtype=dtype)
+            z = zarr.open(
+                str(tmp.name),
+                shape=shape + yx_shape,
+                dtype=dtype,
+                chunks=tuple([1] * len(shape) + yx_shape),  # VERY IMPORTANT FOR SPEED!
```

Comment:
Would it be more optimal to have the shape be something like the length of a z-stack? 

also inthe future with the grouped acq events may be worth revisiting, but I htink this is a good start

Reply:
@ianhi yes maybe! I did some test trying to display a `sequenced` acquisition in real-time and setting this to the shape of a single image gave me the fastest result. But we can definitely do more tests!